repos/fdbt-site/src/interfaces/index.ts

```diff
@@ -734,3 +734,16 @@ export interface Cap {
     durationAmount: string;
     durationUnits: ExpiryUnit;
 }
+
+export interface CapPricePerDistances {
+    distanceFrom: string;
+    distanceTo: string;
+    minimumPrice?: string;
```

Comment:
Why are these optional? We always need a max and min price.

Reply:
only on the first row